definitions/contested/json/CaseField/CaseField-DFR-911-nonprod.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "fdrCaseDocuments",
```

Comment:
I think it would be a good practice if we started naming the collections with a Collection suffix, that looks better when mapping in finrem-common and makes us able to avoid @JsonProperty.